boards/arm/frdm_k64f/pinmux.c

```diff
@@ -104,10 +104,12 @@ static int frdm_k64f_pinmux_init(const struct device *dev)
 
 	pinmux_pin_set(portb,  1, PORT_PCR_MUX(kPORT_MuxAlt4));
 
+#if DT_NODE_HAS_STATUS(DT_CHILD(DT_NODELABEL(enet),  ptp), okay)
```

Comment:
may we can add a config for ENABLE_PTP_EVENT_PINS to conctrol this pinmux, not use the device node

Reply:
I'll vote against yet a config. My solution does it seamless. I can't see how it becomes better by adding a config. 
Most other pinmux entries in the frdm__k64f board are already based on DT_ macros.
Enabling ptp in the device tree without defining pins doesn't really make sense - or does it (Refer you previous comment)?
Hopefully in the (near) future all pinmuxing is found in the dts making issues like this more transparent.

Reply:
I tend to agree that using device tree for this is more inline with the intent of device tree.